examples/iex_wasm/static/script.js

```diff
@@ -0,0 +1,50 @@
+import { Popcorn } from "./wasm/popcorn.js";
+
+const LANGUAGE = document.querySelector('meta[name="code-language"]').content;
+
+var term = new Terminal();
+term.open(document.getElementById(LANGUAGE + '_terminal'));
+window.terminal = term;
```

Comment:
1. Don't use `var`
2. It doesn't make sense to have `setup` function and do init stuff outside of it.